cmake/ExternalLibs.cmake

```diff
@@ -0,0 +1,27 @@
+# RMM
```

Comment:
Can we rename the cmake module to be more rmm specific?

Reply:
Sure.

Reply:
I'm actually removing this CMake module. To be consistent with other plugins, the RMM plugin have been moved to `plugin/CMakeLists.txt`.